test/SpagoSpec.hs

```diff
@@ -766,23 +783,36 @@ spec = around_ setup $ do
   describe "spago bundle-module" $ do
 
     it "Spago should successfully make a module" $ do
+      pursVersion :: Version.SemVer <- getPursVersion
+      purs0_15_0 :: Version.SemVer <- esmVersion
 
       spago ["init"] >>= shouldBeSuccess
       spago ["build"] >>= shouldBeSuccess
       -- Now we don't remove the output folder, but we pass the `--no-build`
       -- flag to skip rebuilding (i.e. we are counting on the previous command
       -- to have built stuff for us)
-      spago ["bundle-module", "--to", "bundle-module.js", "--no-build"] >>= shouldBeSuccess
-      checkFixture "bundle-module.js"
+      if pursVersion >= purs0_15_0 then do
+        spago ["bundle-module", "--to", "bundle-module.js", "--no-build"] >>= shouldBeSuccess
+        checkFixture "bundle-module-esm.js"
```

Comment:
Whoops! Might want to bundle to `bundle-module-esm.js`, not `bundle-module.js`

Reply:
Good catch!